website/templates/website/index.html

```diff
@@ -59,63 +60,45 @@ <h1 class="index-title" style="text-align: center;">WORKSHOPS</h1>
         <hr class="section-heading-spacer">
         <div class="clearfix"></div>
         <h2 class="index-title">Introduction to Game Development</h2>
-        <!-- need to change game name! Don't get copy righted pleasee-->
         <div class="lead">This workshop allows students to unleash their full creative imaginations as we explore how to make our very own versions of games like "Snappy Bird" and “RecBot” </div>
-        <button type="button"><a href="../online_workshops/gamedev/"> Click here to learn!</a></button>
+        <button type="button" class="event-button"><a href="../online_workshops/gamedev/"> Click here to learn!</a></button>
       </div>
     </div>
-  </div>
+    <br></br>
 
-  <div class="container">
+    <!-- Web Dev -->
     <div class="row">
       <div class="col-lg-5 col-md-6 col-sm-6 va">
         <hr class="section-heading-spacer">
         <div class="clearfix"></div>
         <h2 class="index-title">Introduction to Web Development</h2>
         <div class="lead">Students will uncover the true colours of a website behind its pages, pictures and links, as we guide you through all the nooks, crannies and secrets of web design. </div>
-        <button type="button"><a href="../online_workshops/webdev"> Click here to learn!</a></button>
+        <button type="button" class="event-button"><a href="../online_workshops/webdev"> Click here to learn!</a></button>
       </div>
       
       <div class="col-lg-5 offset-lg-2 col-md-6 col-sm-6">
           <img src="{% static 'website/index/webdev.png' %}" alt="Webdev image" class="img-fluid" style="background: linear-gradient(123.24deg, #BD54C7 -0.1%, #25028E 72.24%); border-radius: 2%;">
       </div>
     </div>
-  </div>
+    <br></br>
 
-  <div class="container">
+    <!-- Cyber Security -->
     <div class="row">
-      <div class="col-lg-5 col-md-6 col-sm-6">
-        <img src="{% static 'website/index/ai.png' %}" alt="AI image" class="img-fluid" style="background: linear-gradient(123.24deg, #BD54C7 -0.1%, #25028E 72.24%); border-radius:2%;">
+      <div class="col-lg-5  col-md-6 col-sm-6">
+        <img src="{% static 'website/index/security.png' %}" alt="Security image" class="img-fluid" style="background: linear-gradient(123.24deg, #BD54C7 -0.1%, #25028E 72.24%); border-radius:2%">
```

Comment:
Note to future selves: Again we should probably be dynamically generating this feed...but lower priority